### ISSUE TITLE
Support define(moduleName, callback)

### ISSUE THREAD

**Issue by pago** [jaredhanson/deamdify#6]
There is one more parameter set for the define function that seems to be unsupported at the moment.

An AMD module can be defined by passing its name (string) and the definition callback to the define function. This simply defines a named object without any dependencies.

It'd be great if you could support that as well or mention that it isn't supported.

Let me know if you don't have the time to do it but would like to have that feature. Maybe I could implement it myself and make a pull request for it.

And a sample code:

```javascript
define('jidejs/base/Util', function() {
    return { foo: 'bar' };
});
```

When deamdifyied this will result in a module where the `define` function is still invoked.

**Comment by reggi** [comment]
I just found this out the hard way too.

Demo here:

https://github.com/reggi/browserify-deamdify-test

**Comment by tbranyen** [comment]
Closed via fa415987d48ec9c37b3a2088a0d72cf9393868f7
